guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>Mask R-CNN is a state of the art model for instance segmentation, developed on top of Faster R-CNN. Faster R-CNN is <strong>a</strong> region-based convolutional neural network [2], that returns bounding boxes <strong>for</strong> each object and its class label with <strong>a</strong> confidence score</p><p> </p><p>Stage1: ... to give <strong>a </strong>set of region proposals....</p><p>Region proposals <strong>are</strong> regions in <strong>the </strong>feature map which contain <strong>the </strong>object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>